### DISCUSSION TITLE
Package Manager and proxy

### DISCUSSION THREAD

**Discussion by s-celles** [OpenModelica/OpenModelica#11205] [Q&A]
Hello,

I'm considering using OpenModelica at work with my students but I need to know if proxy settings is currently supported.

I did a quick search https://www.google.com/search?q=openmodelica+proxy+package+manager
and found https://openmodelica.org/images/M_images/OpenModelicaWorkshop_2022/1530_openmodelica2022-sjoelund-package-manager.pdf slide 19 and found Future Work I Better support for proxy servers?

Not sure what it means.

Any idea?

Kind regards

**Comment by casella** [comment]
Can you please be more specific regarding what would you like to access behind a proxy:
- installation files (on which OS)?
- libraries handled by the Package Manager?
- other stuff?
